addon/templates/components/welcome-page.hbs

```diff
@@ -1,10 +1,10 @@
-<div id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
+<main id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
   <div class="columns">
     <div class="tomster">
       <img src="ember-welcome-page/images/construction.png" alt="Under construction">
     </div>
     <div class="welcome">
-      <h2 id="title">Congratulations, you made it!</h2>
+      <h1 id="title">Congratulations, you made it!</h1>
```

Comment:
Happy to merge this line 😉

Reply:
https://github.com/ember-cli/ember-welcome-page/pull/157